.pylintdict

```diff
@@ -1,509 +0,0 @@
-adjoint
```

Comment:
The file .pylintdict contains word exceptions for the spell checker and shouldn't be deleted.

Reply:
I'm sorry, I only deleted the .pylintdict because some commits that I made in another PR come together in this PR. I'll fix this.